config/main.py

```diff
@@ -688,6 +692,10 @@ def _restart_services():
     click.echo("Restarting SONiC target ...")
     clicommon.run_command("sudo systemctl restart sonic.target")
 
+    # Monitor the script of 'container_checker' after restarting services
+    click.echo("Enabling container monitoring ...")
+    clicommon.run_command("sudo monit monitor container_checker")
```

Comment:
Suggest moving this after the Monit reload. Shouldn't make a difference, but it feels like it makes more sense.

Reply:
I checked that moving the command `sudo monit monitor container_checker` after `sudo monit reload` did not make a difference.